Python/Product/Uwp/Microsoft.PythonTools.Uwp.targets

```diff
@@ -30,8 +30,6 @@
 
   <ItemGroup>
     <PtvsdFiles Include="$(PtvsLocation)ptvsd\**\*" />
```

Comment:
The directory name for `$(PtvsLocation)` has changed, layout looks like this:

```
Extensions\Microsoft\Python\
    Core\
    Django\
    IronPython\
    Profiling\
    Uwp\
```

(On the bright side, no version number in the directory anymore.)

The experimental hive will be different again, but that will be easier for you to check yourself right now.